lib/Runtime/Base/JnDirectFields.h

```diff
@@ -644,70 +643,32 @@ ENTRY(toLength)
 ENTRY(toInteger)
 ENTRY(arraySpeciesCreate)
 ENTRY(arrayCreateDataPropertyOrThrow)
-
-// EngineInterfaceObject built-ins
-ENTRY(builtInGlobalObjectEntryIsFinite)
-ENTRY(builtInGlobalObjectEntryIsNaN)
-ENTRY(builtInGlobalObjectEval)
-ENTRY(builtInJavascriptArrayEntryConcat)
+ENTRY(builtInJavascriptArrayEntryIndexOf)
```

Comment:
Why are these 3 special and not included in the header?

Reply:
because the header goes hand in hand with installing JavascriptArray::EntryInfo::IndexOf and the like onto the engine interface object, but since theyre jsbuiltins, the jsbuiltin interface object goes and basically monkey-patches it (if I remember correctly). @sigatrev would know more.